operator/controllers/common/metrics.go

```diff
@@ -98,3 +98,30 @@ func GetActiveInstances(ctx context.Context, client client.Client, reconcileObje
 
 	return res, nil
 }
+
+func GetInstances(ctx context.Context, client client.Client, reconcileObjectList client.ObjectList) ([]common.GaugeValue, error) {
```

Comment:
you are returning a data point here. I would not call this function `GetInstance`.

What metric data point are we returning here? Why a Gauge that alternates between 0 and 1?